pkgs/development/compilers/graalvm/community-edition/hashes.nix

```diff
@@ -1,57 +1,57 @@
-# Generated by ./update.sh script
+# Generated by /home/greg/dev/nixpkgs/pkgs/development/compilers/graalvm/community-edition/update.sh script
```

Comment:
Please run this script from the proper location (inside the ` pkgs/development/compilers/graalvm/community-edition` directory).

Reply:
Or fix the script to use only the basename.

Reply:
Done!